lib/renderer/web-frame-init.ts

```diff
@@ -13,11 +13,6 @@ export const webFrameInit = () => {
   ipcRendererUtils.handle(IPC_MESSAGES.RENDERER_WEB_FRAME_METHOD, (
     event, method: keyof WebFrameMethod, ...args: any[]
   ) => {
-    // TODO(MarshallOfSound): Remove once the world-safe-execute-javascript deprecation warning is removed
-    if (method.startsWith('executeJavaScript')) {
-      return (webFrame as any)[`_${method}`](...args);
-    }
```

Comment:
do we want to keep this in for backportability?

Reply:
Removing this (the deprecation warning) makes the implementation easier.  Need to figure out how best to handle this in older branches you're right

Reply:
I am removing this in https://github.com/electron/electron/pull/28456